client/src/tokens/colors.js

```diff
@@ -183,6 +183,15 @@ const staticColors = {
     },
   },
   info: {
+    125: {
```

Comment:
@thibaudcolas please advise if we want this or not (and if so, review the test changes as well), thank you

Reply:
I’d rather we avoid without consulting with @benenright or @nicklee. The current result looks great in light mode but not in dark mode, this is the kind of thing that should also be reflected in our theme tokens, and ideally we wouldn’t be introducing new variables without it being reflected in the designs.